backend/plugins/payment_api/src/modules/corporateGateway/khanbank/graphql/resolvers/statements.ts

```
@@ -0,0 +1,10 @@
+const KhanbankStatement = {
+  async transactions(statement: any, _params, {}) {
```

ESLint

[error] Unexpected empty object pattern.

(no-empty-pattern)